src/device_analysis.cu

```diff
@@ -198,6 +134,7 @@ void analysis_GPU(bool verbose, float* d_SPS_input, float *h_candidate_list, siz
 	//-------------------------------------------------------------------------
 	//------------ Splitting input data into chunks
 	std::vector<int> DM_list;
+	// TODO: What is logic behind this equation
 	unsigned long int max_timesamples=(free_mem*0.95)/(5.5*sizeof(float) + 2*sizeof(ushort));
```

Comment:
This calculates maximum number of time samples we can fit into available memory based on memory requirements of the SPS per one sample.